repository/flex/metadata.json

```diff
@@ -2,14 +2,14 @@
     "id": "flex",
     "displayName": "Flex",
     "description":  "The 1st Dynamic Marketplace for Starknet",
-    "host": "flexing.gg",
+    "host": "hyperflex.market",
     "categories": ["nft", "mobile"],
     "contracts": [
       {
         "tag": "flex_marketplace",
         "addresses": {
           "mainnet-alpha": [
-            "0x04b1b3fdf34d00288a7956e6342fb366a1510a9387d321c87f3301d990ac19d4"
+            "0x115e92613aa2057fa1ea0e47c6b99faab48e887ad62cb12aa5d2ed01a3066d9"
```

Comment:
pls add the new address to the array instead of removing the old one, we need to track activities over time

Reply:
added in latest commit